src/Analyzer.JsonRuleEngine/Schemas/RuleDefinition.cs

```diff
@@ -35,6 +35,12 @@ internal class RuleDefinition
         [JsonProperty]
         public string HelpUri { get; set; }
 
+        /// <summary>
+        /// Gets or sets the severity of the rule.
+        /// </summary>
+        [JsonProperty]
+        public int Severity { get; set; }
```

Comment:
I also suggest making it nullable, so it's clearer when the severity is not defined in a rule.

Reply:
`public enum Severity { high = 1, medium = 2, low = 3};` 
Something like this^? Where would I set this and I'm assuming the above lines changes to something like this? : 
`[JsonProperty]
        public Severity _Severity { get; set; }`

Reply:
This would be defined in the Types library.  And the property name can remain `Severity` - the compiler will understand the property name is different than the type.

Reply:
I am still unable to get it to work. 
In Types.IEvaluation:
`        /// <summary>
        /// The severity of the rule.
        /// </summary>
        public enum Severity {
            /// <summary>
            /// Severity High = 1
            /// </summary>
            High = 1,
            /// <summary>
            /// Severity Medium = 2
            /// </summary>
            Medium = 2,
            /// <summary>
            /// Severity Low = 3
            /// </summary>
            Low = 3
        };`

In JsonRuleEngine.RuleDefiniton
`        /// <summary>
        /// Gets or sets the severity of the rule.
        /// </summary>
        [JsonProperty]
        public Types.IEvaluation.Severity Severity { get; set; } = Types.IEvaluation.Severity.Medium;`

but, then all the other files complain and I don't have a getter for the enum.

Reply:
@anaismiller I can work with you offline to get this sorted out.

Reply:
Changed, thank you